### DISCUSSION TITLE
Does the 1 click installer installs deepspeed on windows?

### DISCUSSION THREAD

**Discussion by if-ai** [oobabooga/text-generation-webui#736] [Q&A]
I saw people have manage to install the whl's for deepspeed in windows and wonder if is possible to add it on the oneclick if not yet on it?

**Comment by QuantumAlignmentLabOnline** [comment]
Where did you see that?

**Reply by if-ai** [reply]
search deepspeed on the A1111 discussions

**Reply by poisenbery** [reply]
> search deepspeed on the A1111 discussions

https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/6730#discussioncomment-4721245



> I saw people have manage to install the whl's for deepspeed in windows

Are you sure about that???

**Reply by if-ai** [reply]
No, not really sure since I have had both cases of trying a wheel created on another computer and had not success and have to compile them myself but also had the times when I downloaded it and it works. that is why I am asking here. Anyway I look on the site packages it doesn't install deepspeed so I self answered my own question

**Comment by BarfingLemurs** [comment]
gimme that .whl

**Comment by if-ai** [comment]
I look on the site packages it doesn't install deepspeed so I self answered my own question
